definitions/scenarios/upgrade_to_capsule_6_13.rb

```diff
@@ -0,0 +1,92 @@
+module Scenarios::Capsule_6_13
+  class Abstract < ForemanMaintain::Scenario
+    def self.upgrade_metadata(&block)
+      metadata do
+        tags :upgrade_scenario
+        confine do
+          feature(:capsule) &&
+            (feature(:capsule).current_minor_version == '6.12' || \
+            ForemanMaintain.upgrade_in_progress == '6.13')
+        end
+        instance_eval(&block)
+      end
+    end
+
+    def target_version
+      '6.13'
+    end
+  end
+
+  class PreUpgradeCheck < Abstract
+    upgrade_metadata do
+      description 'Checks before upgrading to Capsule 6.13'
+      tags :pre_upgrade_checks
+      run_strategy :fail_slow
+    end
+
+    def compose
+      add_steps(find_checks(:default))
+      add_steps(find_checks(:pre_upgrade))
+      add_step(Checks::Foreman::CheckpointSegments)
+      add_step(Checks::Repositories::Validate.new(:version => '6.13'))
+    end
+  end
+
+  class PreMigrations < Abstract
+    upgrade_metadata do
+      description 'Procedures before migrating to Capsule 6.13'
+      tags :pre_migrations
+    end
+
+    def compose
+      add_steps(find_procedures(:pre_migrations))
+      add_step(Procedures::Service::Stop.new)
+    end
+  end
+
+  class Migrations < Abstract
+    upgrade_metadata do
+      description 'Migration scripts to Capsule 6.13'
+      tags :migrations
+    end
+
+    def set_context_mapping
+      context.map(:assumeyes, Procedures::Installer::Upgrade => :assumeyes)
+    end
+
+    def compose
+      add_step(Procedures::Repositories::Setup.new(:version => '6.13'))
+      modules_to_enable = ["satellite-capsule:#{el_short_name}"]
+      add_step(Procedures::Packages::EnableModules.new(:module_names => modules_to_enable))
```

Comment:
this is *technically* not required, but also absolutely doesn't hurt, so I left it in

(we could also probably move it to `Procedures::Repositories::Setup`, as it feels like it's part of it anyway?